src/dug/core.py

```diff
@@ -55,34 +54,26 @@ class Search:
          * disease->study
          * disease->phenotype->study
     """
+    def __init__(self, cfg=config.from_env(), indices=['concepts_index', 'variables_index', 'kg_index']):
```

Comment:
I don't think we should have a default value for the config here, I think we should make it a required param (and update other usages accordingly)


```suggestion
    def __init__(self, cfg: Config, indices=['concepts_index', 'variables_index', 'kg_index']):
```